examples/pachyderm/sklearn_pachyderm.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>Helm to the rescue! We're getting quite a lot of exampels with helm, it may get to a point where we'll want to centralise the setting up of Helm in our docs to point people similar to how we do for seldon install, maybe would be good to add an issue?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I will open a separate issue on that, that's a good idea!</p><p> </p><p>Minio is becoming very useful and I will be happy to have more examples just being able to depend on it being configured (like seldon-core itself is).</p>

Reply:
https://github.com/SeldonIO/seldon-core/issues/1740